node/public/JS/bmi-calc.js

```diff
@@ -0,0 +1,95 @@
+const status = {
+    UNDERWEIGHT: 'underweight',
+    NORMAL: 'normal',
+    OVERWEIGHT: 'overweight',
+    OBESE: 'obese'
+
+}
+
+function getStatus(bmi) {
```

Comment:
this appears to be a global function can it interfere with other function in different file with same name